src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -213,6 +213,19 @@ public void convertToLocalDatabase() {
         return database.getEntries();
     }
 
+    /**
+     * check if the database has any empty entries
+     * @return true if the database has any empty entries; otherwise false
+     */
+    public boolean hasEmptyEntries() {
+        for (BibEntry currentEntry: this.getEntries()) {
```

Comment:
You can use streams here:
`return this.getEntries().stream().anyMatch(entry->entry.getFields().isEmpty())`

Reply:
Thank you! I have updated the function.